tekton/src/pipelines/gitops/provision-mas-apps.yml.j2

```diff
@@ -869,3 +909,264 @@ spec:
+      when:
+        - input: "$(params.mas_app_channel_predict)"
+          operator: notin
+          values: [""]
+        - input: "$(params.cp4d_wsl_action)"
+          operator: in
+          values: ["install"]
+
+    # 13.1.b Configure WSL in MAS Core
+    - name: gitops-suite-watson-studio-config
+      runAfter:
+        - gitops-cp4d-wsl
+      params:
+
+        {{ lookup('template', pipeline_src_dir ~ '/taskdefs/gitops/common/gitops-params.yml.j2') | indent(8) }}
+
+        {{ lookup('template', pipeline_src_dir ~ '/taskdefs/gitops/common/secrets-params.yml.j2') | indent(8) }}
+        
+        {{ lookup('template', pipeline_src_dir ~ '/taskdefs/gitops/common/git-params.yml.j2') | indent(8) }}
+
+        - name: mas_instance_id
+          value: $(params.mas_instance_id)
+
+        - name: mas_config_scope
+          value: wsapp
+
+        - name: watson_studio_username
+          value: $(params.watson_studio_username)
+        - name: watson_studio_password
+          value: $(params.watson_studio_password)
+        - name: watson_studio_url
+          value: $(params.watson_studio_url)
+        - name: cp4d_wsl_action
+          value: $(params.cp4d_wsl_action)
+
+      taskRef:
+        kind: Task
+        name: gitops-suite-watson-studio-config
+      when:
+        - input: "$(params.mas_app_channel_predict)"
+          operator: notin
+          values: [""]
+        - input: "$(params.cp4d_wsl_action)"
+          operator: in
+          values: ["install"]
+          
+      workspaces:
+        - name: configs
+          workspace: configs
+        - name: shared-gitops-configs
+          workspace: shared-gitops-configs
+
+    # 13.2 CP4D wml
+    - name: gitops-cp4d-wml
+      runAfter:
+        - gitops-cp4d
+      params:
+        {{ lookup('template', pipeline_src_dir ~ '/taskdefs/gitops/common/gitops-params.yml.j2') | indent(8) }}
+
+        {{ lookup('template', pipeline_src_dir ~ '/taskdefs/gitops/common/secrets-params.yml.j2') | indent(8) }}
+        
+        {{ lookup('template', pipeline_src_dir ~ '/taskdefs/gitops/common/git-params.yml.j2') | indent(8) }}
+
+        - name: mas_instance_id
+          value: $(params.mas_instance_id)
+
+        - name: cpd_service_name
+          value: wml
+        - name: cpd_product_version
+          value: $(params.cpd_product_version)
+        - name: cpd_service_block_storage_class
+          value: $(params.cpd_service_block_storage_class)
+        - name: cpd_service_storage_class
+          value: $(params.cpd_service_storage_class)
+
+      workspaces:
+        - name: configs
+          workspace: configs
+        - name: shared-gitops-configs
+          workspace: shared-gitops-configs
+      taskRef:
+        kind: Task
+        name: gitops-cp4d-service
+      when:
+        - input: "$(params.mas_app_channel_predict)"
+          operator: notin
+          values: [""]
+        - input: "$(params.cp4d_watson_machine_learning_action)"
```

Comment:
I think we are missing cp4d_watson_machine_learning_action (and the other cp4d actions other than wsl) from that params